src/probes/crash_probe.c

```diff
@@ -443,12 +444,18 @@ static bool is_banned_path(char *fullpath)
         }
 
         // Anything outside of /usr/, or in /usr/local/, we consider third-party
-        if (g_regex_match_simple("^[!/]usr[!/]", fullpath, 0, 0) == FALSE) {
-                return true;
-        } else if (g_regex_match_simple("^[!/]usr[!/]local[!/]", fullpath,
-                                        0, 0) == TRUE) {
-                return true;
-        }
+
+	if (strncmp(fullpath, "/usr/", 5) != 0)
```

Comment:
also your code is pretty horrible in that it adds at least 16Kb to the size of the final binary and in memory...  we're trying to save, not waste, memory ;-)